src/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { DatePicker, DatePickerProps } from '@mui/x-date-pickers/DatePicker';
+import React from 'react';
+import { Controller } from 'react-hook-form';
+
+type Props = DatePickerProps<string> & {
+  minDate: string;
+  maxDate: string;
+};
```

Comment:
Potrzebujemy `name` każdy input ma swój unikalny `name`
Nie zawsze będzie `label` Select Date np. Start Date, End Date

```suggestion
type Props = DatePickerProps<string> & {
  name: string;
  label: string;
  minDate?: string;
  maxDate?: string;
};
```